v2/fleet-local/control/flight-director@.service

```diff
@@ -2,16 +2,18 @@
 Description=Flight Director @ %i
 # it's implied that chronos & marathon require mesos
 # look here: https://github.com/behance/mesos-systemd/tree/master/v2/fleet
-After=docker.service chronos.service marathon.service
+After=docker.service chronos@%i.service marathon@%i.service
```

Comment:
no, it's more of a `if there is a chronos@%i.service on this machine, schedule this unit to start after it`. What *really* does the blocking is on line 16.